lib/chewy/query.rb

```diff
@@ -818,6 +818,20 @@ def types! *params
       chain { criteria.update_types params, purge: true }
     end
 
+    # Sets <tt>search_type</tt> for request.
+    # For instance, one can use <tt>search_type=count</tt> to fetch only total count of records or to fetch only aggregations without fetching records.
+    #
+    #   scope = UsersIndex.search_type(:count)
+    #   scope.count == 0  # no records actually fetched
+    #   scope.total == 10 # but we know a total count of them
+    #
+    #   scope = UsersIndex.aggs(max_age: { max: { field: 'age' } }).search_type(:count)
+    #   max_age = scope.aggs['max_age']['value']
+    #
+    def search_type val
+      chain { options.merge!(search_type: val) }
```

Comment:
Are you sure about `criteria.update_options`? I think that `Criteria` is just responsible for constructing a `{ body: ... }` part but `search_type` is not a part of body. It's just a query option just like `type` or `index`.
Or you're suggesting to add a new `update_options` method to `Query` ?